src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -17,19 +19,38 @@ import java.util.stream.Collectors
  * creating it, do we assume it exists?
  */
 class SetupEnvironment : CliktCommand(help="create conda environment for PHG") {
-    // The output directory will store the log and error files from the ProcessBUilder conda create command
-    val outputDir by option("-o", "--outputDir", help = "Directory where ProcessBuilder, conda create env command log files will be written")
-        .required()
+
+    private val myLogger = Logger.getLogger("net.maizegenetics.phgv2.cli.SetupEnvironment")
+
+    val envFile by option("-e", "--env-file", help = "File containing the conda environment definition")
```

Comment:
I'm not sure why this is an option?

Reply:
@tcasstevens how should this be coded as an optional parameter?

Reply:
I would have thought not having this parameter at all?

Reply:
We decided to allow them to use their own environment file in case they for some reason needed something additional.  Perhaps a conda environment for 2 things?  @zrm22 what are your thoughts?

Reply:
I think it is helpful for the user to submit their own.  As lynn mentioned we might not have some software installed in our environment that a user may want to include.  By allowing them to submit their own yaml file we can accommodate this.  Most people likely will just do the conda commands themselves though for external env, but it doesn't hurt to allow users to do it within our system.